src/test/java/uk/gov/hmcts/ccd/domain/service/common/SecurityClassificationServiceTest.java

```diff
@@ -487,6 +496,41 @@ void shouldFilterAllFieldsForCaseWithMultiSelectListTypes() throws IOException {
             );
         }
 
+        @Test
```

Comment:
New tests that were missing for SecurityClassificationService to increase branch coverage.